deploy/openshift4/nsx-ipi-helper.yaml

```diff
@@ -0,0 +1,49 @@
+# nsx-ipi-helper DaemonSet
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nsx-ipi-helper
+  namespace: nsx-system-operator
+  labels:
+    tier: nsx-networking
```

Comment:
No need to add `nsx-networking` and `v1` lebal, or maybe we can just remove `labels`?

Reply:
OK, I just copy it from nsx-node-agent, in fact I also think it's not useful... will remove the labels.